python/ray/data/datasource/file_meta_provider.py

```diff
@@ -398,7 +398,9 @@ def _fetch_metadata_parallel(
     # dominates the Ray task overhead while ensuring good parallelism.
     # Always launch at least 2 parallel fetch tasks.
     parallelism = max(len(uris) // desired_uris_per_task, 2)
-    metadata_fetch_bar = ProgressBar("Metadata Fetch Progress", total=parallelism)
+    metadata_fetch_bar = ProgressBar(
+        "Metadata Fetch Progress", total=parallelism, unit="file"
```

Comment:
I think this unit is task and not file?

Reply:
i also initially thought so as well, but looked into this further, and i think it is files. when `metadata_fetch_bar.fetch_until_complete()` is called, it increments progress based on the number of outputs yielded from the tasks, which i believe will correspond to 1 per input file. Let me know if that sounds incorrect @bveeramani

Reply:
> it increments progress based on the number of outputs yielded from the tasks

It updates based on the number completed tasks, right? Each task returns a list of file metadata: https://github.com/bveeramani/ray/blob/ab5b7c3b238a688802dcc87399cc3b6afbbc5603/python/ray/data/_internal/progress_bar.py#L92-L98

Also, the total for the progress bar is the number of tasks: `max(len(uris) // desired_uris_per_task, 2)`.

Reply:
That makes sense to me. I'll commit the change @scottjlee @bveeramani

Reply:
ah yeah, i overlooked the total part -- you're right, should be # of tasks. @bt2513 could you update this to `task`?

Reply:
done! ✔️